Content.Server/PDA/PDASystem.cs

```diff
@@ -211,5 +227,9 @@ private void AfterUIOpen(EntityUid uid, PDAComponent pda, AfterActivatableUIOpen
 
             return address;
         }
+
+        private EntityUid? GetStationUid(PDAComponent pda) {
```

Comment:
why make this a function of its own its just 1 line
also you should probably make everything take the pda uid instead of adding .Owner to things